test/e2e/resourceinterpreter_test.go

```diff
@@ -121,4 +123,74 @@ var _ = ginkgo.Describe("Resource interpreter webhook testing", func() {
 			framework.RemovePropagationPolicy(karmadaClient, policy.Namespace, policy.Name)
 		})
 	})
+
+	ginkgo.Context("Workload resource ReviseReplica hook testing", func() {
```

Comment:
How about rename this context to `InterpreterOperation ReviseReplica testing`. That way, it's consistent with the above.

Reply:
Updated.